.github/workflows/docker.yml

```diff
@@ -0,0 +1,68 @@
+name: Build Docker Image
+
+on:
+    push:
+
+jobs:
+    docker-compose:
+        name: Build and Excute Tests on Docker Compose
+        runs-on: ubuntu-latest
+        env:
+            COMPOSE_FILE: docker-compose.test.yml
+        
+        steps:
+            -   name: Checkout code
+                uses: actions/checkout@v2
+
+            -   name: Build docker images
+                run: docker-compose build
+
+            -   name: Run tests
+                run: docker-compose run ci
+    
+    docker:
+        runs-on: ubuntu-latest
+        name: Build and Excute Tests on Docker Containers
+        steps:
+            -
+                name: Checkout
+                uses: actions/checkout@v2
+            -
+                name: Set up Docker Buildx
+                id: buildx
+                uses: docker/setup-buildx-action@v1
+            -
+                name: Build Docker Image
+                run: |
+                    docker build .docker --tag sylius_app
+            -
+                name: Check Versions
+                run: |
+                    chmod +x .docker/run-tests.sh
+                    ./.docker/run-tests.sh
+            -
+                name: Shutdown default MySQL
+                run: sudo service mysql stop
+            -
+                name: Setup MySQL
+                run: |
+                    docker network create sylius_network
+                    docker run \
+                    --platform linux/amd64 \
+                    -e MYSQL_ALLOW_EMPTY_PASSWORD=true \
+                    -p 3306:3306/tcp \
+                    -v "$(pwd)"/.docker/dev/mysql:/docker-entrypoint-initdb.d:delegated \
+                    --name mysql \
+                    --net sylius_network \
+                    -d mysql:5.7
+            -
+                name: Run PHPUnit
```

Comment:
The last step of this config is calling ci from Makefile, so if I'm not mistaken, following operation will be executed:
```bash
	composer install --no-interaction --no-scripts
	bin/console sylius:install --no-interaction
	bin/console sylius:fixtures:load default --no-interaction
	yarn install --pure-lockfile
	node_modules/gulp/bin/gulp.js
	vendor/bin/phpunit
	vendor/bin/phpspec run --ansi --no-interaction -f dot
	vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="@javascript&&~@todo&&~@cli"  # CLI Behat
	vendor/bin/behat --colors --strict --no-interaction -vvv -f progress --tags="~@javascript&&~@todo&&~@cli" # NON JS Behat
```

therefore this step name is not the most fortune I guess